versions.txt

```diff
@@ -5,4 +5,3 @@ gax:2.7.0:2.7.1-SNAPSHOT
 gax-bom:2.7.0:2.7.1-SNAPSHOT
 gax-grpc:2.7.0:2.7.1-SNAPSHOT
```

Comment:
I guess we will be able to remove these two once we get rid of Gradle, right?

Reply:
To be clear, I should be pointing to `gax-grpc` and `gax-httpjson`. `gax-bom` is being used in the canned template in `gax-bom/pom.xml`.

Reply:
I think so. Especially, if these versions will always move in sync.